pynio/block.py

```diff
@@ -37,6 +48,47 @@ def name(self):
     def type(self):
         return self._type
 
+    @property
+    def template(self):
+        return self._template
+
+    @template.setter
+    def template(self, value):
+        # do type checking later
+        self._template = value
+
+    @property
+    def config(self):
+        return self._config
+
+    @config.setter
+    def config(self, value):
+        if self._template is None:
+            self._config = value
+            return
+        config = deepcopy(self._template)
+        config.readonly = False
+        config.update(value, drop_unknown=True,
```

Comment:
This doesn't seem to be working for lists. I tested with a Simulator block and it's not loading the attributes.